pom.xml

```diff
@@ -71,6 +72,18 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-project</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
+      <version>${configuration-as-code.version}</version>
```

Comment:
```suggestion
```
I think those versions are already defined by parent pom

Reply:
I should have paid more attention to the error I got, missed that I declared the dependency with that version. I've pushed up a change removing the specific version now.